docs/tutorials/remote_soc.rst

```diff
@@ -0,0 +1,209 @@
+        input clk0,
+        input csb0,
+        input web0,
+        input [3:0] wmask0,
+        input [8:0] addr0,
+        input [31:0] din0,
+        output reg [31:0] dout0,
+    // Port 1: R
+        input clk1,
+        input csb1,
+        input [8:0] addr1,
+        output reg [31:0] dout1
+      );
+    endmodule
+
+Next, you need to create a top-level Verilog module containing one ``picorv32`` CPU core, one ``sky130_sram_2k`` memory, and signal wiring to connect their I/O ports together. Note that for the sake of brevity, this module does not include some optional parameters and signals. Check `our picorv32_ram example <https://github.com/siliconcompiler/siliconcompiler/blob/main/examples/picorv32_ram/picorv32_top.v>`_ for a more complete ``picorv32_top`` declaration::
+
+    `timescale 1 ns / 1 ps
+
+    module picorv32_top (
+            input clk, resetn,
+            output reg trap,
+
+            // Look-Ahead Interface
+            output            mem_la_read,
+            output            mem_la_write,
+            output     [31:0] mem_la_addr,
+            output reg [31:0] mem_la_wdata,
+            output reg [ 3:0] mem_la_wstrb,
+
+            // Pico Co-Processor Interface (PCPI)
+            output reg        pcpi_valid,
+            output reg [31:0] pcpi_insn,
+            output     [31:0] pcpi_rs1,
+            output     [31:0] pcpi_rs2,
+            input             pcpi_wr,
+            input      [31:0] pcpi_rd,
+            input             pcpi_wait,
+            input             pcpi_ready,
+
+            // IRQ Interface
+            input      [31:0] irq,
+            output reg [31:0] eoi,
+
+            // Trace Interface
+            output reg        trace_valid,
+            output reg [35:0] trace_data
+    );
+
+        // Memory signals.
+        reg mem_valid, mem_instr, mem_ready;
+        reg [31:0] mem_addr;
+        reg [31:0] mem_wdata;
+        reg [ 3:0] mem_wstrb;
+        reg [31:0] mem_rdata;
+
+        // No 'ready' signal in sky130 SRAM macro; presumably it is single-cycle?
+        always @(posedge clk)
+            mem_ready <= mem_valid;
+
+        // (Signals have the same name as the picorv32 module: use '.*' to autofill)
+        picorv32 rv32_soc (
+          .*
+        );
+
+        // SRAM with always-active chip select and write control bits.
+        sky130_sram_2kbyte_1rw1r_32x512_8 sram (
+            .clk0(clk),
+            .csb0('b0),
+            .web0(!(mem_wstrb != 0)),
+            .wmask0(mem_wstrb),
+            .addr0(mem_addr),
+            .din0(mem_wdata),
+            .dout0(mem_rdata),
+            .clk1(clk),
+            .csb1('b1),
+            .addr1('b0),
+            .dout1()
+        );
+    endmodule
+
+Finally, your core build script will need to be updated to import the new SRAM Library, and specify some extra parameters such as die size and macro placement::
+
+    import siliconcompiler
+
+    design = 'picorv32_top'
+    die_width = 1000
```

Comment:
Later is referenced as `die_w`

Reply:
Good catch, thanks - I think `width`/`height` are probably better for a tutorial since they're more descriptive than `w`/`h`.